junit-platform-console/src/main/java/org/junit/platform/console/options/Theme.java

```diff
@@ -79,43 +79,43 @@ public static Theme valueOf(Charset charset) {
 		this.blank = new String(new char[vertical().length()]).replace('\0', ' ');
 	}
 
-	public String root() {
+	public final String root() {
```

Comment:
@marcphilipp As it turns out, `veritical()` _can_ be overridden! (Thanks for @cushon over at https://github.com/google/error-prone/issues/709#issuecomment-323873109 for pointing this out to me.)

If one writes the following code:
```java
public enum Theme {
  ASCII(".", "| ", "+--", "'--", "[OK]", "[A]", "[X]", "[S]") {
    @Override public String vertical() { ... }
  } ,
  ...
}
```

Then one is effectively overriding `Theme.vertical()` from within the `ASCII` constant. So it seems to me that ("ConstructorInvokesOverridable")[http://errorprone.info/bugpattern/ConstructorInvokesOverridable] is working correctly.

Thus we at the very least must either make `Theme.vertical()` final or disable the rule (which seems to be the least wise option to me).